### ISSUE TITLE
Handle multipolygon geometry type

### ISSUE THREAD

**Issue by FarhodKurbonov** [humangeo/leaflet-dvf#66]
I have data in geoJSON format which includes 'MultiPolygon' geometries. Leaflet's L.geoJson handle multipolygon layer  'as one polygon', so when I mouse over one polygon, all other polygons which are part of multipolygon will change  style. But L.choroplethDataLayer binds mouse event to each polygon separately(each polygon in multipolygon). 

In consequence of this behavior, when I set text on each layer, all MultiPolygon layers which include many polygons also insert text

Thanks for your suggest and sorry for my bad Enlish

I hope photo would help understand my mind:

![mape](https://cloud.githubusercontent.com/assets/8686969/9068430/de3183e4-3afc-11e5-861f-9a5de4ffe700.png)

**Comment by sfairgrieve** [comment]
@FarhodKurbonov this issue makes sense.  Let me look into it and get back to you.

**Comment by sfairgrieve** [comment]
@FarhodKurbonov is this occurring in Leaflet 0.7.3 or 1.0.0 beta?  Just wondering, b/c I think Leaflet 1.0.0 beta treats a MultiPolygon as a single Polygon (based on checking out the source code), which I think would not have this problem.  I'm working on a fix for 0.7.3 as well.

**Comment by FarhodKurbonov** [comment]
Yes, it is occurring in Leaflet 0.7.3. My problem, resolved after updated to 1.0.0 beta. Thank you for your help.

**Comment by sfairgrieve** [comment]
If you use the 1.0dev branch, that should be compatible with 1.0.0 beta 1.  Let me know if you run into any issues.
